README.md

```diff
@@ -163,280 +179,41 @@ const order: MarketOrderSpec = {
-  setup,
-  teardown,
-});
-```
-
-[Check the full example](./examples/advanced/setup-and-teardown.ts)
-
-<!--
-TODO:
- ### Market scan
-
-You can scan the market for available providers and their offers. This is useful when you want to see what's available
-before placing an order.
-
-```ts
-await glm.market.scan(order).subscribe({
-  next: (proposal) => {
-    console.log("Received proposal from provider", proposal.provider.name);
-  },
-  complete: () => {
-    console.log("Market scan completed");
-  },
-});
-```
-
-[Check the full example](./examples/basic/market-scan.ts) -->
-
-## Going further
-
-If you wish to learn more about how the SDK functions under the hood, please check out our more advanced examples:
-
-- [Creating pools manually](./examples/advanced/manual-pools.ts)
-- [Performing all market operations manually](./examples/advanced/step-by-step.ts)
-- [(for library authors) Override internal module](./examples/advanced/override-module.ts)
-
-## More examples
+## Examples
 
 The [examples directory](./examples) in the repository contains various usage patterns for the SDK. You can browse
 through them and learn about the recommended practices. All examples are automatically tested during our release
 process.
 
-In case you find an issue with the examples, feel free to submit
-an [issue report](https://github.com/golemfactory/golem-js/issues) to the repository.
-
 You can find even more examples and tutorials in
 the [JavaScript API section of the Golem Network Docs](https://docs.golem.network/docs/creators/javascript).
 
+## Documentation
+
+Visit our [official documentation](https://docs.golem.network/docs/creators/javascript) to learn more about the
+JavaScript SDK and how to use it.
+
 ## Debugging
 
-The SDK uses the [debug](https://www.npmjs.com/package/debug) package to provide debug logs. To enable them, set the `DEBUG` environment variable to `golem-js:*` or `golem-js:market:*` to see all logs or only the market-related ones, respectively. For more information, please refer to the [debug package documentation](https://www.npmjs.com/package/debug).
+The SDK uses the [debug](https://www.npmjs.com/package/debug) package to provide debug logs. To enable them, set
+the `DEBUG` environment variable to `golem-js:*` or `golem-js:market:*` to see all logs or only the market-related ones,
+respectively. For more information, please refer to
+the [debug package documentation](https://www.npmjs.com/package/debug).
 
 ## Testing
 
-Read the dedicated [testing documentation](./TESTING.md) to learn more about how to run tests of the SDK.
+Read the dedicated [testing documentation](./docs/TESTING.md) to learn more about how to run tests of the SDK.
 
 ## Contributing
 
-It is recommended to run unit tests and static code analysis before committing changes.
+Read the [Contributing Guide](./CONTRIBUTING.md) for details on how you can get involved. In case you find an issue with the examples or the SDK itself, feel free to submit
```

Comment:
Why is `CONTRIBUTING.md` in root when all other markdown files are in `docs/`?

Reply:
Good hint, I thought that it's the pattern recommended by GH but after checking the best practices I see that this could be placed in the docs as well.

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes